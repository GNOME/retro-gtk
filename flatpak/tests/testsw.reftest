[Retro Reftest]
Path=/testsw
Core=/app/lib/libretro/testsw_libretro.so

[Frame 0]
State=Refresh
Video=testsw.png
