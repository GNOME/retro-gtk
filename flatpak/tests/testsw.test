[Retro Reftest]
Path=/testsw
Core=/app/lib/libretro/testsw_libretro.so

[Frame 0]
Tests=Run;Video;StateRefresh;
Video=testsw.png
