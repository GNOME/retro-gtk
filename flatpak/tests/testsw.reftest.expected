/testsw/Boot: OK
/testsw/Options: OK
/testsw/0/State Refresh: OK
/testsw/0/Run: OK
/testsw/0/Video: OK
